CHANGELOG/CHANGELOG-4.2.md

```diff
@@ -0,0 +1,30 @@
+# Release notes for v4.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v4.1.1
+
+## Changes by Kind
+
+### Feature
+
+- Logging for gRPC call can now be limited configurably with flag `--max-grpc-log-length` ([#411](https://github.com/kubernetes-csi/external-attacher/pull/411), [@leiyiz](https://github.com/leiyiz))
+
+## Dependencies
+
+### Added
+- cloud.google.com/go/compute: v1.12.1
```

Comment:
although it's auto-generated, I don't see this in go.mod so not sure where this is coming from...

Reply:
It got pulled by `google.golang.org/grpc@v1.52.3`. I did not dig deeper why grpc depends on google cloud libs.